tests/flow/test_overall.py

```diff
@@ -196,7 +196,7 @@ def test_insert(self):
         with env.assertResponseError():
             env.cmd('bf.insert', 'missingFilter', 'ERROR', 'big')
         with env.assertResponseError():
-            env.cmd('bf.insert', 'missingFilter', 'EXPANSION', '0', 'ITEMS', 'foo')
+            env.cmd('bf.insert', 'missingFilter', 'EXPANSION', '-1', 'ITEMS', 'foo')
```

Comment:
I am curious what exactly required this change when we needed only to change the configuration options-related code?

Reply:
0 expansions is (now) valid. and equivalent to NONSCALING